data/json/items/generic/bedding.json

```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "id": "pillowcase",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "pillowcase" },
+    "looks_like": "pillow",
+    "description": "A simple cotton pillowcase.",
+    "weight": "415 g",
+    "volume": "1 L",
+    "price": 0,
+    "price_postapoc": 2,
+    "to_hit": -5,
+    "material": [ "cotton" ],
+    "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "15 L", "max_contains_weight": "15 kg", "moves": 400 } ],
```

Comment:
Pillows are currently ~1L so 15L seems too large for a pillow case.

Reply:
pillows are soft and can be squashed into a smaller space, when the foam/stuffing is contracted it's much larger.
Pillows should probably be at least 3L however.

Reply:
Though I do agree _pillows_ should be 3L, but this is for pillow **cases.** A pillowcase all scrunched up is easily 1L.
15L is less than you'd think, only about 4 gallons.

Reply:
Additionally I made them the same size as canvas sacks (15L) because I thought they seemed about the right size, for additional usefulness, and because it's my first time doing this.